test/site/_collections/runEleventy.js

```diff
@@ -0,0 +1,14 @@
+const assert = require('assert');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+
+module.exports = async function(env) {
+  console.log('Running Eleventy...');
+  try {
+    await exec(
+      `ELEVENTY_ENV=${env} npx @11ty/eleventy --config="./eleventy.js"`,
```

Comment:
npx should pick up eleventy from our node_modules dir, so should be compatible with our main run. If I run `npm run eleventy` here, it is run from the / root dir i picks up the evelenty.js config located there, even if I provide the correct path later on (which is weird). I can also run directly `../../../../../node_modules/.bin/eleventy --config="./eleventy.js" ` but slightly prefer npx as it will get dependencies in case some modules are missing. Wdyt?